app/src/services/lecture.services.ts

```diff
@@ -0,0 +1,44 @@
+import axios from "axios";
+
+// Interfaces
+import { Lecture } from "../interfaces/Lecture";
+
+export const client = axios.create({
+  baseURL: import.meta.env.VITE_BACKEND_URL,
+  withCredentials: true,
+  responseType: 'json',
+  timeout: 30000,
+});
+
+
+// Send the info to lecture service
+const addLecture = async ({title, description}: Lecture, token: string, sid: string) => {
+  //console.log(client.post(`/api/lectures/create/${sid}`))
+  console.log(import.meta.env.VITE_BACKEND_URL + `/api/lectures/create/${sid}`)
+  return await axios.post(
+    `${import.meta.env.VITE_BACKEND_URL}/api/lectures/create/${sid}`,
```

Comment:
Do not have actions in endpoint

should be api/lectures/:id

Also should it be PUT instead of POST? Any thoughts

Reply:
We'll fix it to put.
There is an endpoint, it's just in our backend pull request.